ethereum/contracts/BasicSendChannel.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
+pragma experimental ABIEncoderV2;
+
+import "./SendChannel.sol";
+
+// BasicSendChannel is a basic channel that just sends messages with a nonce.
+contract BasicSendChannel is SendChannel {
+    uint256 public currentNonce;
```

Comment:
It would be good to have consistent naming style for global state across our codebase. Looking through other crypto code, it seems people just use the short and sweet name `nonce` to refer to the current cryptographic nonce for a global thing.

Examples where `nonce` is used to name the current nonce of a global account: 
* https://substrate.dev/rustdocs/v2.0.1/evm/backend/struct.Basic.html#structfield.nonce
* https://substrate.dev/rustdocs/v2.0.1/frame_system/struct.AccountInfo.html#structfield.nonce
* https://github.com/ethereum/go-ethereum/blob/66c3eb2f1a5c6bb4b5a678ce819f9984ac09e246/core/state/state_object.go#L103

Similarly, in our Substrate code, I'm not using `CurrentBalance` to track the current balance of a global account.

Reply:
this is not important tbh, resolving